From b60ef5f1c6bf652b025ba169b7fe8e37539e7625 Mon Sep 17 00:00:00 2001
From: Jia Wang <wangjia@ultrarisc.com>
Date: Fri, 15 May 2026 09:17:58 +0800
Subject: [RUYI PATCH] FROMLIST: dt-bindings: riscv: cpus: Add UltraRISC CP100
 compatible

Update Documentation for supporting UltraRISC CP100 based CPU.

CP100 is used in UltraRISC DP1000 SoC.

Signed-off-by: Jia Wang <wangjia@ultrarisc.com>
Acked-by: Conor Dooley <conor.dooley@microchip.com>
Link: https://lore.kernel.org/r/20260515-ultrarisc-pinctrl-v1-2-bf559589ea8a@ultrarisc.com
Signed-off-by: Han Gao <gaohan@iscas.ac.cn>
---
 Documentation/devicetree/bindings/riscv/cpus.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/riscv/cpus.yaml b/Documentation/devicetree/bindings/riscv/cpus.yaml
index 5feeb22030..9f52267177 100644
--- a/Documentation/devicetree/bindings/riscv/cpus.yaml
+++ b/Documentation/devicetree/bindings/riscv/cpus.yaml
@@ -67,6 +67,7 @@ properties:
               - thead,c908
               - thead,c910
               - thead,c920
+              - ultrarisc,cp100
           - const: riscv
       - items:
           - enum:
-- 
2.54.0

